File: 3412-PermutationDifferenceBetweenTwoStrings/3412-PermutationDifferenceBetweenTwoStrings.py
# Last updated: 12/28/2025, 4:52:08 PM
class Solution:
    def findPermutationDifference(self, s: str, t: str) -> int:
        s1=[];l=[]
        m=0
        for i in s:
            s1.append(i)
        for i in t:
            l.append(i)
        for i in s1:
            for j in l:
                if(i==j):
                    m+=abs(s1.index(i)-l.index(j))
                
        # print(s1,l)
        return m

File: 3412-PermutationDifferenceBetweenTwoStrings/3412-PermutationDifferenceBetweenTwoStrings.java
// Last updated: 12/28/2025, 4:52:04 PM
public class Solution {
    public int findPermutationDifference(String s, String t) {
        char[] s1 = s.toCharArray();
        char[] l = t.toCharArray();
        int m = 0;
        
        for (char i : s1) {
            for (char j : l) {
                if (i == j) {
                    m += Math.abs(new String(s1).indexOf(i) - new String(l).indexOf(j));
                }
            }
        }

        return m;
    }
}
